en.txt

```diff
@@ -0,0 +1,61 @@
+History:
+	I mostly have experience with linux (servers, desktops), unix, c programming
+	language
+	I like to discover new, innovative solutions for various IT problems
+	I like to use interesting open-source solutions, I am looking for interesting ones
+	ideas, and I try to implement them myself, or sometimes improve them.
+	I have worked briefly in several startups. I am active in cooperation.
+	What I did in startups:
+	- start-up 1: I wrote Python automation scripts for various tasks (mostly
+	related to web-scrap).
+	Some real examples:
+	- The bot is designed to find bicycles of a specific price, in particular locations.
+	When found, send messages to Telegram.
+	- The bot is designed to find the most popular Reddit posts, and according to the topic, make 10 min
+	video montage to upload them to different YouTube channels
+	- start-up 2: I worked with embedded Linux in the yacto project.
+	I was selling WordPress installation services.
+
+Languages:
+	English Excellent
+	English Lithuanian
+	Russian On average
+
+
+Igudžiai:
+	 Linux (Unix) server administration
+		bash scripting,
+		Work automation,
+		System logging
+		Knowledge of server architecture,
+		Web Application Installation (Like WordPress)
+		Experience with AWS cloud architecture (AWS cloud architecture)
+		Scalable server/cloud infrastructure
+		Administration of BSD, and MacOS systems
+
+
+Scripting languages:
+	Python,
+	bash,
+	Javascript.
+
+Back-end
+	MySQL (Database Installation, Configuration, Monitoring)
+	Note.js (and other native javascript runtimes like deno.js)
+	PHP routers
+	Apache2
+	Nginx
+
+Web'ai/Front-end:
+	Plain HTML, CSS, Javascript, WordPress, PHP
+	Front-end Frameworks like Astro.js, react.js, php Symfony
+
+Java
+	I worked with a Java professional who assigned me Java tasks
+	Assignments were reviewed, corrected, and completed.
+	You can see the entire tasking process in the GitHub links below
+		https://github.com/Adomo-Software/rating-system
+		https://github.com/Adomo-Software/caesars-palace-casino
+		https://github.com/Adomo-Software/java-fix-me
+		https://github.com/Adomo-Software/hotel
+		https://github.com/Adomo-Software/StringMatch_2
```

Comment:
Very important points:
1. How long the learning took?
2. What you have learned on each of the mentioned task/repos above?